core/doc/snippets/uServices-modulecontext/main.cpp

```diff
@@ -16,9 +16,7 @@ void RetrieveModuleContext()
 //! [GetModuleContext]
 
 //! [InitializeModule]
-#include <usModuleInitialization.h>
 
-US_INITIALIZE_MODULE
 //! [InitializeModule]
```

Comment:
The comment should stay since I mistakenly removed

    #include <usModuleInitialization.h>
    US_INITIALIZE_MODULE

which I believe should remain in since it illustrates the connection between declaring a bundle and getting its bundle context.